### DISCUSSION TITLE
点燃工业TNT造成的崩溃

### DISCUSSION THREAD

**Discussion by liuhaoyu-China** [IndustrialCraftMC/IndustrialCraft2#21] [【漏洞反馈】Bugs Report]
放下工业TNT并用打火石点燃后，游戏有TNT的斯斯声，但画面立刻卡死，约3秒后显示保存世界然后闪退
闪退后原存档无法进入，加载时直接卡在0%未响应
版本：forge43.2.10，PCL启动器，除工业2外没有其他模组
[错误报告-2023-5-6_0.47.11.zip](https://github.com/AbbottMc/IndustrialCraft2/files/11408459/-2023-5-6_0.47.11.zip)

**Comment by liuhaoyu-China** [comment]
更正一下，再次进存档就正常了，当时测的时候应该是电脑卡死了

**Comment by liuhaoyu-China** [comment]
不确定是不是电脑太辣鸡导致的

**Reply by Y400050** [reply]
不是的这是概率事件，我上次反映的时候具体测试了的，测了10次点燃的时候必定会崩溃，重进的时候有2次崩溃

**Comment by WaterDouble** [comment]
我认为大概率应该是cpu带不动突然之间的大爆炸了，本人以前也经历过（悲）

**Reply by liuhaoyu-China** [reply]
我觉得不是，因为普通TNT的威力甚至比工业TNT还要大，但是普通TNT不会闪退，可能还是有BAG或者优化太差

**Comment by Y400050** [comment]
我之前就已经反映了，看了还没修好

**Comment by WaterDouble** [comment]
个人实验证实这个bug确确实实是存在的，而且比较严重。本机崩溃后可以正常进入且正常显示其爆炸带来的掉落物结果，所以看起来应该是性能优化问题而导致的，也请官方大大重视一下这个bug。

**Reply by AbbottMc** [reply]
后续会修复的，目前正在进行声音系统的重构~

**Comment by AbbottMc** [comment]
该内容与已有的Bug反馈重复，详情见：https://github.com/AbbottMc/IndustrialCraft2/discussions/8#discussion-5159733

**Reply by Y400050** [reply]
6
